### DISCUSSION TITLE
"For you" shows seemingly completely random content

### DISCUSSION THREAD

**Discussion by rsms** [community/community#34475] [Other Feature Feedback, Questions, & Ideas] [answered]
Even with filtering, it shows projects which are for software I don't work with and from people I don't follow. None of the top five projects listed had _any_ relevance for my work or interests.

For example, I have never heard of this person before. I don't follow him or use any of the projects he's dealing with. Totally zero affinity as far as I know:

<img width="619" alt="Screen Shot 2022-09-28 at 08 54 25" src="https://user-images.githubusercontent.com/47046/192827524-5395b518-ca01-4ab7-9889-4eef02e6556d.png">

Like [@davetron5000 says in another post](https://github.com/orgs/community/discussions/21086), I'm not even sure what "For you" is supposed to do (a short description at the top would be nice), but I'm guessing it is like "Explore"

<img width="622" alt="Screen Shot 2022-09-28 at 08 48 12" src="https://user-images.githubusercontent.com/47046/192826009-1b9e68b0-1f08-443c-b853-bc4e83de5a9a.png">

(I found [an existing "question" that has been marked as "answered"](https://github.com/orgs/community/discussions/21086) but adding another "answer" to that doesn't seem like the right thing to do, so making a new post.)

**Comment by mikekavouras** [comment, marked as answer]
@rsms we just (today) released an option to mark content as "not relevant." Hopefully this helps!

https://github.blog/changelog/2022-10-06-express-disinterest-on-for-you-feed-rollup-items

**Reply by rsms** [reply]
Thank you!

**Reply by rn4n** [reply]
Funny that, even though some users' content is marked as "not relevant", some time later that same content appears again in my feed. As an example, imagine an user called Foo:

1. Foo starred a repo, cool;
2. Me, marked that content as "I'm not interessed in starred repositories from Foo", cool;
3. Two (or more) days lates, Foo appears again in my feed after star a repo. 

🤡 LOL, it's totally annoying.

**Comment by syahrulbagas** [comment]
though like that. maybe this can be used to get to know each other, who knows it can contribute to our project, even though it's annoying.
